transformers/test_data/deal.go

```diff
@@ -58,15 +57,15 @@ var dealModel = event.InsertionModel{
 	TableName:  constants.DealTable,
 	OrderedColumns: []event.ColumnName{
 		event.HeaderFK,
-		deal.Id,
-		constants.AddressColumn,
+		event.AddressFK,
 		event.LogFK,
+		constants.BidIDColumn,
 	},
 	ColumnValues: event.ColumnValues{
-		event.HeaderFK:          DealHeaderSyncLog.HeaderID,
-		constants.BidIDColumn:   "10000000000000000",
-		constants.AddressColumn: rawDealLog.Address.Hex(),
-		event.LogFK:             DealHeaderSyncLog.ID,
+		event.HeaderFK:        DealHeaderSyncLog.HeaderID,
+		event.AddressFK:       rawDealLog.Address.Hex(),
```

Comment:
I think that most of the tests that use these models for assertions, we look up the address id in the test itself rather than relying on this hex string. So maybe we should remove this from the test model since it's not an accurate representation of what a model will look like similar to how we're handling it for [test dent](https://github.com/makerdao/vdb-mcd-transformers/blob/staging/transformers/test_data/dent.go#L80)?

Reply:
Nice catch! Yeah that hex definitely doesn't belong there.